ConcernsCaseWork/ConcernsCaseWork.Service/Decision/DecisionService.cs

```diff
@@ -99,7 +97,7 @@ public async Task<GetDecisionOutcomeResponse> GetDecisionOutcome(long decisionId
 				BusinessAreasConsulted = new List<DecisionOutcomeBusinessArea> { DecisionOutcomeBusinessArea.BusinessPartner, DecisionOutcomeBusinessArea.Funding }
 			};
 
-			_logger.LogInformation($"Retrieved decision {outcomeId} for decisionID {decisionId}");
+			_logger.LogInformation($"Retrieved decision outcome {outcomeId} for decisionID {decisionId}");

```

Comment:
DecisionService.GetDecisionOutcome seems to be hard coded to return test data

Reply:
So I think its just a placeholder, for when the real API is available, which is in review

Reply:
OK. Maybe for something like that we at least comment, or preferably put a #If Development around it. Probably better to fail on staging/prod than pretend to work